build.gradle.kts

```diff
@@ -56,6 +59,7 @@ dependencies {
     testCompile("junit:junit:4.12")
     testCompile("org.assertj:assertj-core:3.11.1")
     testCompile("org.hamcrest:hamcrest-library:1.3")
+    testCompile("com.atlassian.performance.tools:ssh-ubuntu:0.1.0")
```

Comment:
[0.2.0](https://github.com/atlassian/ssh-ubuntu/compare/release-0.1.0...release-0.2.0) should work better